### ISSUE TITLE
Can't watch my own videos in anything above 720p

### ISSUE THREAD

**Issue by Primeey** [YePpHa/YouTubeCenter#2079]
I don't know why I'm bothering posting this because the dev seems to not care for the script anymore

Currently I can't watch any video **I upload** in above 720p. When I disable the script I can watch in in the source quality. It is such a dumb issue but it's real, watch my video showing it.

https://www.youtube.com/watch?v=wvut4tHMY7M

**Comment by Yonezpt** [comment]
This is a limitation of disabling DASH playback, see: https://github.com/YePpHa/YouTubeCenter/wiki/Missing-480p-and-1080p-resolutions

The solution is to enable DASH in order to get the other quality options.

**Comment by Pazns** [comment]
Indeed, this is a limitation on Youtube side. Video data without DASH technology is available only for 720 and 360p quality. Maybe below 360p too, I'm not sure.

**Comment by Primeey** [comment]
DASH is enabled.

**Comment by Yonezpt** [comment]
I can confirm that DASH is failing when enabled, looks like YTC is throwing an error.

**Comment by gram2222** [comment]
My video too, can only watch at 360p and 720p as my Dash playback already enabled

https://www.youtube.com/watch?v=6dUsvXmWww4

**Comment by Azure505** [comment]
Also confirming this issue on my end. The only thing I've noticed that might even be remotely helpful in diagnosing the cause of this issue is that I've never noticed it happen on a webm video, only mp4, though not *every* mp4 video. It's very odd. 
  
https://www.youtube.com/watch?v=B28a0H77VBw  
  
There's a quick test video which, as with the rest of the videos I've uploaded in the past, has this issue.
